app/view_models/font_awesome_icon.rb

```diff
@@ -0,0 +1,70 @@
+class FontAwesomeIcon
+  def self.graphical(attrs = {})
+    new(attrs.merge(graphical: true))
+  end
+
+  def self.labelled(attrs = {})
+    new(attrs.merge(graphical: false))
+  end
+
+  def initialize(type:, label:, style: :normal, fixed_width: false, graphical:)
```

Comment:
I think that `style` shouldn't have a default and we should have to be explicit each time.

Reply:
Cool. Agree as well :)